src/main/java/mchorse/mappet/api/scripts/code/entities/ScriptEntity.java

```diff
@@ -308,7 +313,11 @@ public IScriptEntity getTarget()
     @Override
     public void setTarget(IScriptEntity entity)
     {
-        if (this.entity instanceof EntityLiving && entity != null && entity.isLivingBase())
+        if (entity == null)
+        {
+            ((EntityLiving) this.entity).setAttackTarget(null);
```

Comment:
Possible ClassCastException. Before it was checked if `this.entity instanceof EntityLiving`, now you directly cast to that class although it is not certain what type `this.entity` is.

Reply:
Thanks :sweat_smile:

Reply:
No problem 👍